ceph-dev-new-setup/build/build

```diff
@@ -103,12 +103,11 @@ echo current version $cephver
 
 srcdir=`pwd`
 
+setup_container_runtime
 if command -v podman; then
-    PODMAN=podman
-    PODMAN_STORAGE_DIR="$HOME/.local/share/containers/storage/"
-    test -d $PODMAN_STORAGE_DIR && sudo chgrp -R $(groups | cut -d' ' -f1) $PODMAN_STORAGE_DIR
+  PODMAN=podman
 else
-    PODMAN="sudo docker"
+  PODMAN=docker
```

Comment:
how do we no longer need sudo with docker?

Reply:
did we need it in the first place?

Reply:
it used to be the case that only root could contact dockerd, sometime in the past

Reply:
...or the user needs to be part of the docker group, seems to still be the case

Reply:
It checks for the group now